odoo/sql_db.py

```diff
@@ -240,7 +240,11 @@ def execute(self, query, params=None, log_exceptions=None):
             res = self._obj.execute(query, params)
         except Exception as e:
             if self._default_log_exceptions if log_exceptions is None else log_exceptions:
+<<<<<<< HEAD
                 _logger.error("bad query: %s\nERROR: %s", self._obj.query.decode() or query, e)
+=======
+                _logger.error("bad query: %s\nERROR: %s", tools.ustr(self._obj.query or query), e)
```

Comment:
This line seems good.

Reply:
As in, the issue was already fixed and we can just close the forward-port PR?